### PR TITLE
Fix typecheck

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Fix typecheck in /web & /mobile. 

Few issues:
1. Force type of keyv https://github.com/DefinitelyTyped/DefinitelyTyped/issues/62793#issuecomment-1285167695
2. Update @audius/sdk versions in the right places & update everything to 1.3.10
3. Fix common import in common buy $AUDIO selectors

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

npm run typecheck

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/common/src/store/ui/buy-audio/selectors.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { CommonState } from '../../../store/reducers'
+import { CommonState } from 'store/reducers'
```
Comment (raymondjacobson) [inline packages/common/src/store/ui/buy-audio/selectors.ts:1]: This was necessary to stop seeing stuff like this

```
src/components/buy-audio-modal/components/InProgressPage.tsx:160:37 - error TS2538: Type 'undefined' cannot be used as an index type.

160             ? providerBasedMessages[buyAudioProvider].purchaseError
                                        ~~~~~~~~~~~~~~~~
```

**Comment by dylanjeffers** [comment]
@raymondjacobson i think @michellebrier figured out a solution here, just wanted to double check why it seemed to fail again?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-fix-typecheck

**Comment by raymondjacobson** [comment]
> @raymondjacobson i think @michellebrier figured out a solution here, just wanted to double check why it seemed to fail again?

Ah yes. Excellent. Reverted the keyv changes I made in this PR

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-fix-typecheck

**Review by sliptype — APPROVED** [review]
~Typecheck was already passing on ci, was this something you are seeing locally?~ Oh gotcha this was on the release branch

**Pull request merged**